azure-storage-queue/src/Queue/QueueRestProxy.php

```diff
@@ -413,7 +413,7 @@ public function createMessageAsync(
             $body,
             $options
         )->then(function ($response) use ($dataSerializer) {
-            $parsed = $dataSerializer->unserialize($response->getBody());
+            $parsed = $dataSerializer->unserialize((string) $response->getBody());
```

Comment:
`$response->getBody()` returns a Stream object not a string, the act of casting it causes the entire stream from beginning to end to be captured according to: https://stackoverflow.com/questions/30549226/guzzlehttp-how-get-the-body-of-a-response-from-guzzle-6